### DISCUSSION TITLE
Documentation to customize template color

### DISCUSSION THREAD

**Discussion by cavasinf** [kevinpapst/TablerBundle#42] [Q&A] [answered]
Hey @kevinpapst,

I was wondering if it was up to us to:
- provide an example how to override default template colors.
- redirect dev to Tabler documentation to find how to override template colors.

I've done mine, I can provide it as a README/example. 
But if we are forwarding the dev to Tabler, it is not needed.

**Comment by kevinpapst** [comment, marked as answer]
Any doc that we provide means less questions by other devs, so I am always pro docs 😄

**Reply by cavasinf** [reply]
Ok 👍 
Opening issue about `navbar-dark` vs `navbar-light` choice then.

**Comment by cavasinf** [comment]
I have my file ready for the template color override.
But I have no idea how I should put that in your git.

Readme ? With explication paragraph ? Should I put the whole file ? or only blocks ?
Help me 😭

**Reply by kevinpapst** [reply]
I have a lot of docs that need to be updated from AdminLTE to Tabler. 
But to show you an example, I added a link from the doc index to a new page.
Just create a new page and if it can be merged with an existing chapter, I will merge those later on.
But for now having g a new page sounds fine to me.

**Reply by kevinpapst** [reply]
Updated index: https://github.com/kevinpapst/TablerBundle/blob/main/docs/index.md
New page: https://github.com/kevinpapst/TablerBundle/blob/main/docs/layout.md
